internal/provider/data_source_scope.go

```diff
@@ -0,0 +1,101 @@
+package provider
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/hashicorp/boundary/api"
+	"github.com/hashicorp/boundary/api/scopes"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const ()
```

Comment:
Lets remove this if we don't need it

Reply:
I've used this now since we are adding in ParentScopeId.